node-module-system/calculate.js

```diff
@@ -0,0 +1,26 @@
+import sum from './add.js';
+import difference from './subtract.js';
+import product from './multiply.js';
+import quotient from './divide.js';
+
+const firstArg = Number(process.argv[2]);
+const secondArg = Number(process.argv[4]);
+
+switch (process.argv[3]) {
```

Comment:
It would be nicer to have all your user input handling in one place. If you declare a variable for `process.argv[3]` and put it up top, the code will read nicer.